### ISSUE TITLE
Mismatched anonymous define() module

### ISSUE THREAD

**Issue by marinsicavirdol** [esvit/ng-table#461]
I'm trying to use ngTable with require.js and  head.js but I'm getting the following error:

_Uncaught Error: Mismatched anonymous define() module: function (angular) {
            return factory(angular);
        }_

Here is my require config:

```javascript
(function(head){
    'use strict';
    head.js(
        { jquery: "../vendor/jquery/dist/jquery.js"},
        { require: "../vendor/requirejs/require.js" },
        { angularjs: "../vendor/angular/angular.js" },
        { uiRouter: "../vendor/angular-ui-router/release/angular-ui-router.js" },
        { angularTable: "../vendor/ng-table/ng-table.js"}
    ).ready("ALL", function(){
        require.config({
            appDir: '',
            baseUrl: '',
            paths: {
            },
            shim: {
            }
        });

        require(['config'], function(config){
        });
    });
}(window.head));
```
Can anyone help me with this problem?

**Comment by vance** [comment]
+1

**Comment by vance** [comment]
yay, it closed

**Comment by james-daddies** [comment]
Know this closed but still shows up in a search result, so instead of crushing the brief glimmer of hope, I found a lead to the issue based upon a SO answer.

http://stackoverflow.com/questions/26992356/ng-table-mismatched-anonymous-define-module/27778108#27778108

TL;DR - Load this library with RequireJS or fork it and remove the define() call from the src.
